rnaseq_dee2_mouse/non_secret_env.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+export WORKSPACE_ID=6583
```

Comment:
Add differen workspace for each data set. Made the workspace and give the access to saksham